pylabnet/launchers/servers/staticline_nidaqmx.py

```diff
@@ -2,10 +2,11 @@
 
 import socket
 
-import pylabnet.hardware.ni_daqs.nidaqmx_card as nidaqmx
+# import pylabnet.hardware.ni_daqs.nidaqmx_card as nidaqmx
 import pylabnet.hardware.staticline.staticline as staticline
 
 from pylabnet.network.client_server.staticline import Service, Client
+import pylabnet.network.client_server.abstract_device as abstract_device
```

Comment:
I don't think you have actually included the `abstract_device` servers in the directory, which means this dummy example can not be run. I suggest either adding `abstract_device` to the servers directory, or using the `dummy=True` flag in order to instantiate devices as indicated here: https://github.com/lukingroup/pylabnet/pull/122

Reply:
This file is no longer in use as we no longer require the implementation of a device-specific Statcline server, and so this file can be deleted. Instead, we only need the normal device server to be running in order for the Staticline GUI to communicate with the device.,